lib/stats/stats-cluster.c

```diff
@@ -26,6 +26,25 @@
 #include <assert.h>
 #include <string.h>
 
+GPtrArray *stats_types;
+static const gchar *module_names[];
+
+void
+stats_cluster_init(void)
+{
+  g_assert(!stats_types);
+  stats_types = g_ptr_array_new_with_free_func(g_free);
+  for (int i = 0; i < SCS_MAX; i++)
```

Comment:
Use `ARRAY_SIZE(module_names)` instead of `SCS_MAX`.

Reply:
I would like to come back to this question if we conclude that I need to bring back the static array, as in the current version this code is eliminated. Also I think I can reason why SCS_MAX is better here, but I do not want to open that topic if we do not need this code anyway.